src/admin/collectionsOrBundles/helpers/collection-filters.ts

```diff
@@ -189,6 +189,10 @@ export function generateCollectionWhereObject(
 		andFilters.push({
 			channel_type: { _neq: excludeChannelType },
 		});
+	} else if (excludeChannelType === null) {
+		andFilters.push({
+			channel_type: { _ilike: '%' },
+		});
```

Comment:
This query conveys more clearly what your are trying to do:
```suggestion
		andFilters.push({
			channel_type: { _is_null: false },
		});
```

Reply:
Amai, da'k daar heb overgekeken in hasura...